src/ParachainRegistry.sol

```diff
@@ -38,14 +41,21 @@ contract ParachainRegistry is IRegistry {
     /// @dev Register parachain, along with index of Tellor pallet within corresponding runtime.
     /// @param _paraId uint32 The parachain identifier.
     /// @param _palletInstance uint8 The index of the Tellor pallet within the parachain's runtime.
-    function register(uint32 _paraId, uint8 _palletInstance) external {
```

Comment:
Please add missing @param function signature comments for the newly added parameters.

e.g.
// @param _weighToFee uint32 Description here.

Also, Tellors convention is to use _param for function parameters as they are coming from the outside, whereas param is something internal to the contract.